arm-software/embedded/CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
 All notable changes to this project will be documented in this file.
+This release is migrated from [LLVM-ET](https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm).
+The package structure remains the same, making this a direct successor release.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
```

Comment:
For the main branch, could you please add the Unreleased section as in LLVM ET https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm/blob/f6f879ed45ff5727d821f6cd29571ee2baf31779/CHANGELOG.md?plain=1#L7

This section should be used for adding changes as they land on the main branch.